UI/window-basic-filters.cpp

```diff
@@ -36,9 +36,18 @@
 #include <QMenu>
 #include <QVariant>
 
-#ifdef _WIN32
-#define WIN32_LEAN_AND_MEAN 1
-#include <Windows.h>
+#if defined(_M_ARM64)
+	#ifdef _WIN32
+		#ifndef WIN32_LEAN_AND_MEAN
+		#define WIN32_LEAN_AND_MEAN 1
+		#endif
+	#endif
+	#include <Windows.h>
```

Comment:
This should be inside the previous definition, but in general this whole thing seems pointless

Reply:
Thanks for pointing that out. It was a mistake, I've made the necessary corrections now.